graphql/src/main/java/com/scalar/db/graphql/datafetcher/QueryGetDataFetcher.java

```diff
@@ -0,0 +1,70 @@
+package com.scalar.db.graphql.datafetcher;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import com.scalar.db.api.Consistency;
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedTransaction;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Result;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.exception.transaction.TransactionException;
+import com.scalar.db.graphql.schema.TableGraphQlModel;
+import com.scalar.db.io.TextValue;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Map;
+import java.util.Optional;
+
+public class QueryGetDataFetcher extends DataFetcherBase<Map<String, Map<String, Object>>> {
+
+  public QueryGetDataFetcher(DistributedStorage storage, TableGraphQlModel tableModel) {
+    super(storage, tableModel);
+  }
+
+  @SuppressWarnings("unchecked")
+  @Override
+  public Map<String, Map<String, Object>> get(DataFetchingEnvironment environment)
+      throws Exception {
+    Map<String, Object> getInput = environment.getArgument("get");
+    Map<String, Object> key = (Map<String, Object>) getInput.get("key");
+    Get get =
+        new Get(createPartitionKeyFromKeyArgument(key), createClusteringKeyFromKeyArgument(key))
+            .forNamespace(tableModel.getNamespaceName())
+            .forTable(tableModel.getTableName());
+    String consistency = (String) getInput.get("consistency");
+    if (consistency != null) {
+      get.withConsistency(Consistency.valueOf(consistency));
+    }
+
+    ImmutableMap.Builder<String, Object> builder = ImmutableMap.builder();
+    performGet(environment, get)
+        .ifPresent(
+            result -> {
+              for (String fieldName : tableModel.getFieldNames()) {
+                result
+                    .getValue(fieldName)
+                    .ifPresent(
+                        value -> {
+                          if (value instanceof TextValue) {
+                            builder.put(fieldName, value.getAsString().get());
```

Comment:
Actually `TextValue` (and `BlobValue` as well) is nullable, so we need to handle the situation `value.getAsString().isPresent() == false`.

Reply:
Great point! I have confirmed that the graphql-java accepts an `Optional` as a data fetcher result, so I have updated the code to keep the `Value.get()` as is. a32b79a 
It's also good from the point of view that it can be set in `ImmutableMap`.